CascadingSelectsDemo/Helpers/GamesHelper.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace CascadingSelectsDemo
+{
+    public static class GamesHelper
+    {
+        public static string[] SelectGames(string developer, string console)
+        {
+            string[] games = new[] { "" };
+            if (developer.Equals("james", StringComparison.OrdinalIgnoreCase))
+            {
+                if (console.Equals("Playstation 3", StringComparison.OrdinalIgnoreCase))
+                    games = new[] { "Fifa", "Madden", "Mario" };
```

Comment:
With the games declaration and initialization above, re-allocation the "games" variable would be pointless.

Each one of these routines could better be served by simply returning the array of games:

``` csharp
if (console.Equals("{developer name}"))
    return new[] { "{first game}", "{second game}", "{third game}" };
```